podpac/datalib/gfs.py

```diff
@@ -55,27 +85,35 @@ def sources(self):
             "date": self.date,
             "hour": self.hour,
             "cache_ctrl": self.cache_ctrl,
-            "s3": self.s3,
         }
-        return np.array([GFSSource(forecast=forecast, **params) for forecast in self.forecasts])
+        return np.array([GFSSourceRaw(forecast=forecast, **params) for forecast in self.forecasts])
 
-    def get_coordinates(self):
-        nc = self.sources[0].coordinates
+    @cached_property
+    def source_coordinates(self):
         base_time = datetime.datetime.strptime("%s %s" % (self.date, self.hour), "%Y%m%d %H%M")
         forecast_times = [base_time + datetime.timedelta(hours=int(h)) for h in self.forecasts]
-        tc = Coordinates(
-            [[dt.strftime("%Y-%m-%d %H:%M") for dt in forecast_times]], dims=["time"], crs=nc.crs, validate_crs=False
+        return Coordinates(
+            [[dt.strftime("%Y-%m-%d %H:%M") for dt in forecast_times]], dims=["time"], validate_crs=False
```

Comment:
No CRS? Seems dangerous.

Reply:
It's just time, does it really matter? I only put it in there in the first place so that `merge_dims` wouldn't complain. But I can put it back in!

Reply:
I missed that, sorry. Yes it's fine with just time. I thought these were the "native" coordinates, I missed it was the source coordinates.